src/main/resources/META-INF/plugin.xml

```diff
@@ -240,72 +240,59 @@
-                        class="com.sourcegraph.cody.edit.actions.EditCancelAction"
+                <action id="cody.fixup.codelens.cancel"
+                        class="com.sourcegraph.cody.edit.actions.lenses.EditCancelAction"
                         text="Cancel Edit"
                         description="Cancel the fixup">
                     <override-text place="GoToAction" text="Cody: Cancel Edit"/>
+                    <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt X" keymap="$default"/>
+                    <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt X" keymap="Mac OS X 10.5+"/>
                 </action>
 
-                <action id="cody.inlineEditRetryAction"
-                        class="com.sourcegraph.cody.edit.actions.EditRetryAction"
+                <action id="cody.fixup.codelens.retry"
+                        class="com.sourcegraph.cody.edit.actions.lenses.EditRetryAction"
                         text="Retry Edit"
                         description="Retry the fixup">
-                    <!-- Key shortcut is handled by cody.editCodeAction, which converts to a Retry if Accept is showing. -->
+                    <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt R" keymap="$default"/>
+                    <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt R" keymap="Mac OS X 10.5+"/>
                     <override-text place="GoToAction" text="Cody: Retry Edit"/>
                 </action>
 
-                <action id="cody.inlineEditUndoAction"
-                        class="com.sourcegraph.cody.edit.actions.EditUndoAction"
+                <action id="cody.fixup.codelens.undo"
+                        class="com.sourcegraph.cody.edit.actions.lenses.EditUndoAction"
                         text="Undo Edit"
                         description="Undo the fixup">
                     <override-text place="GoToAction" text="Cody: Undo Edit"/>
+                    <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt BACK_SPACE" keymap="$default"/>
+                    <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt BACK_SPACE"
+                                       keymap="Mac OS X 10.5+"/>
                 </action>
 
-                <action id="cody.editShowDiffAction"
-                        class="com.sourcegraph.cody.edit.EditShowDiffAction"
+                <action id="cody.fixup.codelens.diff"
+                        class="com.sourcegraph.cody.edit.actions.lenses.EditShowDiffAction"
                         text="Show Diff"
                         description="Show a diff view of the fixup">
                     <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt D" keymap="$default"/>
                     <keyboard-shortcut replace-all="true" first-keystroke="ctrl alt K" keymap="Mac OS X 10.5+"/>
                     <override-text place="GoToAction" text="Cody: Show Diff"/>
                 </action>
```

Comment:
There seems to be some conflicts with existing shortcuts. We'll check with @danielmarquespt today :-)

Reply:
Fixed. I consulted changes with @danielmarquespt